test/extensions/load_balancing_policies/ring_hash/ring_hash_lb_test.cc

```diff
@@ -106,6 +106,16 @@ INSTANTIATE_TEST_SUITE_P(RingHashPrimaryOrFailover, RingHashLoadBalancerTest,
                          ::testing::Values(true, false));
 INSTANTIATE_TEST_SUITE_P(RingHashPrimaryOrFailover, RingHashFailoverTest, ::testing::Values(true));
 
+TEST_P(RingHashLoadBalancerTest, ChooseHostBeforeInit) {
```

Comment:
Personal nit: refactoring PRs should only perform the code-refactor, and not add new functionality. This helps when looking at the historical PRs while trying to understand why some feature was added/updated.
As these are tests, it is less of an issue, but just something to consider for the next time.

Reply:
Got it. This refactoring moved some code from `common/upstream` to `extensions`. This result in a coverage failure.
These new tests is used to improve the coverage ratio.

Reply:
That makes sense, thanks.
I was missing the context around the new tests.